numpy/_core/tests/test_mem_policy.py

```diff
@@ -46,6 +46,16 @@ def get_module(tmp_path):
              Py_DECREF(secret_data);
              return old;
          """),
+        ("set_wrong_capsule_name_data_policy", "METH_NOARGS", """
+             PyObject *secret_data =
```

Comment:
I used the same `struct` for the secret_data_policy. Is this ok or should I create a struct just for this test?

Reply:
That's can be anything here really.  But let's rename the object (will just apply the suggestion though).